pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -460,7 +460,7 @@ func (lbaas *LbaasV2) createLoadBalancer(service *v1.Service, name string, inter
 
 // GetLoadBalancer returns whether the specified load balancer exists and its status
 func (lbaas *LbaasV2) GetLoadBalancer(ctx context.Context, clusterName string, service *v1.Service) (*v1.LoadBalancerStatus, bool, error) {
-	loadBalancerName := cloudprovider.GetLoadBalancerName(service)
+	loadBalancerName := cloudprovider.DefaultGetLoadBalancerName(service)
```

Comment:
This should call `lbaas. GetLoadBalancerName()`

Reply:
Ok, just saw this. Will update this as well.